src/implementations/twig/components/table/table.html.twig

```diff
@@ -100,11 +102,14 @@
     {% if _caption is not empty %}
     <caption class="ecl-table__caption">{{ _caption }}</caption>
     {% endif %}
+
+    {% set _index = 1 %}
     <thead class="{{ _head_css_class }}">
       {% for header in _headers %}
         <tr class="{{ _row_css_class }}">
           {% for cell in header %}
-            {% set _heading_attribute = '' %}
+            {% set _heading_attribute = 'id="' ~ _id  ~ '-' ~ _index ~ '"' %}
```

Comment:
confusing, is this intentional that we first set the _heading_attribute as the id and then if rowspan is set we delete it and recreate it?

Reply:
this was a mistake